src/tests/rest-calls-200-404.spec.ts

```diff
@@ -0,0 +1,59 @@
+import axios from 'axios';
+import { AxiosHarTracker } from '../axios-har-tracker'
+import { writeFileSync } from 'fs';
+
+describe('Check axios-har-tracker for status 200 and 404', () => {
+
+    let axiosTracker, getHar
```

Comment:
Don't use higher scope variable `getHar` for generated har, it shouldn't be shared across specs because it can produce side effects in tests. It can be defined separately in each `it` statement or cleaned each time in `beforeEach`